examples/advanced/Tutorial3/macro/run_sim.C

```diff
@@ -28,7 +32,7 @@ void run_sim(Int_t nEvents = 100, TString mcEngine = "TGeant3")
     gSystem->Setenv("CONFIG_DIR", tut_configdir.Data());
 
     // create Instance of Run Manager class
-    FairRunSim *fRun = new FairRunSim();
+    auto fRun = std::make_unique<FairRunSim>();
```

Comment:
* How does this not conflict with the fact that [`FairRunSim` wants to be a singleton](https://github.com/FairRootGroup/FairRoot/blob/b0482233554eb7cc8ecc81bea588bfcb0e5a6572/fairroot/base/steer/FairRunSim.cxx#L68-L72)?
* And if it would no longer be a singleton, why not just put it on the stack?

Reply:
Side note: If we touch that line, let's also rename the variable (probably in a separate commit). The `f` prefix suggests a member variable which it is clearly not.

Reply:
Well, you know, it's not a classic singleton. It's one that can be deallocated (by the owner). There's just a global pointer to it, for all those, who don't have a pointer to it.

All that said, I didn't want to touch every single line referencing it… but still deallocate it.

But you're right: Let's turn it into a stack variable (and as I need to touch every line anyway, let's a a rename in the same commit).